consensus/src/member.rs

```diff
@@ -70,8 +71,10 @@ enum Task<H: Hasher, D: Data, S: Signature> {
     CoordRequest(UnitCoord),
     // Request parents of the unit with the given hash and Recipient.
     ParentsRequest(H::Hash, Recipient),
-    // Broadcast the given unit.
+    // Broadcast the given unit. This happens only once when the unit is created.
     UnitMulticast(UncheckedSignedUnit<H, D, S>),
```

Comment:
I think the existence of this task is controversial as it is performed only once and the idea of `Task` was rather to capture recurring tasks. Instead of implementing all the handling of this one-shot task, it's probably better to just send a broadcast right away in `on_create` and forget about it :P

Reply:
:+1: